### ISSUE TITLE
Special character in 890 interpreted incorrectly

### ISSUE THREAD

**Issue by boriseetgerink** [SubtitleEdit/subtitleedit#1781]
For a subtitle conversion workflow (890 to WebVTT) I am using portable SubtitleEdit with great results, but lately ran into a problem:

I have an 890 file that contains a [dagger character](https://en.wikipedia.org/wiki/Dagger_(typography)): `ru†ineren`. When this 890 file is opened in [FAB Subtitling](http://www.fab-online.com/eng/subtitling/) it is interpreted correctly as `ruïneren`. The same happens for `†e`, which should be displayed as `ë`. In SubtitleEdit it just displays the dagger character and the next character as it is in the 890 file. Apparently the dagger character should be interpreted as placing a [diaeresis or trema](https://en.wikipedia.org/wiki/Diaeresis_(diacritic)) on the next letter, but I can't find any examples of that online.

The problem is: I've opened SubtitleEdit in many of the common encodings (UTF-8, Windows 1250, Windows 1252, US-ASCII), but that doesn't change the interpretation of the dagger character.

When looking at the binary of the 890 file the dagger character is hex `0x86`. This is indeed the dagger character in the [Windows-1252 encoding](https://en.wikipedia.org/wiki/Windows-1252).

So my questions are:

- Is there anyone here who knows about this convention?
- Are you, the maintainer of the project, alright with applying this convention?

**Comment by boriseetgerink** [comment]
A bit more information about the dagger character: When looking at the binary of the 890 file the dagger character is hex 0x86. This is indeed the dagger character in the Windows-1252 encoding.

**Comment by boriseetgerink** [comment]
I can answer some of my questions after looking at `Nikse.SubtitleEdit.Core.SubtitleFormats.Cavena890`:

890 already does character substitution for certain characters with diacritics. However, for characters with diaeresis (or trema), the substitution isn't complete:

```csharp
text = text.Replace(encoding.GetString(new byte[] { 0x86, 0x41 }), "Ä");
text = text.Replace(encoding.GetString(new byte[] { 0x86, 0x61 }), "ä");
text = text.Replace(encoding.GetString(new byte[] { 0x86, 0x4F }), "Ö");
text = text.Replace(encoding.GetString(new byte[] { 0x86, 0x6F }), "ö");
```

Based on my 890, these lines should be extended like this:

```csharp
text = text.Replace(encoding.GetString(new byte[] { 0x86, 0x45 }), "Ë");
text = text.Replace(encoding.GetString(new byte[] { 0x86, 0x65 }), "ë");
text = text.Replace(encoding.GetString(new byte[] { 0x86, 0x49 }), "Ï");
text = text.Replace(encoding.GetString(new byte[] { 0x86, 0x69 }), "ï");
text = text.Replace(encoding.GetString(new byte[] { 0x86, 0x55 }), "Ü");
text = text.Replace(encoding.GetString(new byte[] { 0x86, 0x75 }), "ü");
```

Source of the character hex numbers: https://en.wikipedia.org/wiki/Windows-1252

I will create a pull request for this change.

**Comment by boriseetgerink** [comment]
If approved, will this fix be included in release 3.4.13 #1774 ? Thanks!

**Comment by niksedk** [comment]
Closed via https://github.com/SubtitleEdit/subtitleedit/commit/6945ccd209b026c0f957fab8dcea35c4a289928f + https://github.com/SubtitleEdit/subtitleedit/commit/9b81bc046be7333f31deffd7e90a3f1e8b5851cd
